README.md

```diff
@@ -1,2 +1,68 @@
+
 # react-native-shopify
-React Native bridge to Shopify Buy SDK
+
+## Getting started
+
+`$ npm install react-native-shopify --save`
+
+### Including Mobile-BUY-SDK
+
+Include the Shopify Mobile Buy SDK in your project to make it available to the bridge. 
+Follow the instructions on their Github page to get started. For example, 
+the recommended and easiest way for iOS is to install it as a Pod. This project will look 
+for headers in the Pods directory. 
```

Comment:
What if SDK is added manually to the project? If it wouldn't work in that case, I would rather write instructions how to install it through the cocoapods.